test/Jsr292/src/com/ibm/j9/jsr292/LookupInTests.java

```diff
@@ -61,13 +71,11 @@
 	 */
 	@Test(groups = { "level.extended" })
 	public void testPublicLookup() throws Throwable {
-		Lookup publicLookup = MethodHandles.publicLookup();
-		assertClassAndMode( publicLookup, Object.class, MethodHandles.Lookup.PUBLIC );
+		assertClassAndMode(localPublicLookup, Object.class, MethodHandles.Lookup.PUBLIC);
+		Assert.assertEquals(localPublicLookup.lookupModes(), MethodHandles.Lookup.PUBLIC);
```

Comment:
`static Lookup PUBLIC_LOOKUP = new Lookup(Object.class, Lookup.PUBLIC | Lookup.UNCONDITIONAL);`

Won't this test fail as the lookupModes() includes UNCONDITIONAL as well?

Reply:
test/Jsr292/src/com/ibm/j9/jsr292/LookupInTests.java only works for Java 8 (829) and Java 9 with b148 in which case there is no UNCONDITIONAL bit added.  The test cases with MODULE/UNCONDITIONAL bit updated were placed at test/OpenJ9_Jsr_292_API/src/com/ibm/j9/jsr292/LookupInTests.java